lib/volcanic/authenticator/v1/principal.rb

```diff
@@ -6,8 +6,8 @@ module Volcanic::Authenticator
   module V1
     # Principal api
     class Principal < CommonPrincipalIdentity
-      attr_reader :created_at, :updated_at, :active
-      attr_accessor :id, :secure_id, :name, :dataset_id, :role_ids, :privilege_ids
+      attr_reader :created_at, :updated_at, :active, :login_attempts
```

Comment:
login_attempts.... should not be here.... it should not be exposed by the API either